data/zilliqa/assets/zil1qldr63ds7yuspqcf02263y2lctmtqmr039vrht/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "ZilWall Paint",
+  "symbol": "ZPAINT",
+  "web": "https://zilwall.com",
+  "whitepaper": "https://zilwall.com/whitepaper",
```

Comment:
also

PRIVATE SELL -> PRIVATE SALE
RESEVERED FUNDS -> RESERVED FUNDS

there might be some other mistakes but since these two are section titles it would be better to fix in priority

Reply:
Oh, Thank you. Fixed :)

Reply:
![1624823495](https://user-images.githubusercontent.com/6033345/123557571-707f3700-d746-11eb-9466-2288e3349327.png)

should replace private sell to private sale here on both the title and the text ;)

Reply:
yep, thank you again. we will make some updates for our whitepaper anyway soon